packages/tupaia-web/src/components/Modal.tsx

```diff
@@ -57,12 +59,14 @@ export const Modal = ({ children, isOpen, onClose }: ModalProps) => {
       onClose={onClose}
       PaperComponent={Paper}
       fullScreen={fullScreen}
-      disablePortal
+      // disablePortal
     >
-      <CloseButton onClick={onClose} color="default">
-        <CloseIcon />
-      </CloseButton>
-      <Wrapper id="overlay-wrapper">{children}</Wrapper>
+      <Wrapper id="overlay-wrapper">
+        <CloseButton color="default">
+          <CloseIcon onClick={onClose} />
```

Comment:
The `onClick` here needs to be on the `CloseButton` as opposed to the `CloseIcon`